style.css

```diff
@@ -0,0 +1,76 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+ul {
+  list-style: none;
+}
+
+header {
+  position: fixed;
+  top: 0;
+  background-color: #fff;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.profile {
+  color: #172b4d;
+  font-weight: bold;
+  font-size: 45px;
+  font-family: "poppins", sans-serif;
+  line-height: 50px;
+}
+
+.dev {
+  font-family: "poppins", sans-serif;
+  margin-top: 12px;
+  font-size: 16px;
+  line-height: 23px;
+  color: #344563;
+}
+
+.connect {
+  margin-top: 15px;
+  color: #7f8cff;
+}
+
+main {
+  background-image: url(./images/header-shapes-mobile.png);
+  background-repeat: no-repeat;
+}
+
+section ul {
+  display: flex;
+  gap: 20px;
+  margin-top: 12px;
+}
+
+.pix {
+  height: 10px;
+}
+
+section {
+  margin: 128px 24px 136px 24px;
+}
+
+header a {
+  margin-top: 58px;
+  margin-left: 24px;
+  font-family: "poppins", sans-serif;
+  font-weight: bold;
+  font-size: 18px;
+}
+
+.fas {
+  margin-top: 58px;
+  margin-right: 24px;
+  color: blue;
+}
```

Comment:
- [ ] Please follow the same design for the hamburger button as the given design.

_Look up the comment about the given design template._